api/src/main/scala/dx/api/DxApi.scala

```diff
@@ -954,6 +954,28 @@ case class DxApi(version: String = "1.0.0", dxEnv: DXEnvironment = DxApi.default
     }
   }
 
+  // Copy object to destination project, if it isn't already there.
+  def cloneObject(id: String,
```

Comment:
Question -- I eventually want the ability to clone both files and applets to the current project, if not already there (see https://jira.internal.dnanexus.com/browse/APPS-752 , https://jira.internal.dnanexus.com/browse/APPS-754), it seems like projectClone can take objects of different types, so could it be done this way? @jdidion